packages/raystack/components/drawer/drawer.module.css

```
@@ -18,7 +18,10 @@
 }
 
 .backdrop[data-ending-style] {
-  transition-duration: calc(var(--drawer-swipe-strength, 1) * 400ms);
+  transition-duration: calc(
+    var(--drawer-swipe-strength, 1) *
```

Stylelint (17.14.0)

[error] Unexpected newline after "*" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)